From 3dee861b0c7ce16325652891bf52cb80771ac679 Mon Sep 17 00:00:00 2001
From: Szymon Mielczarek <szymonx.mielczarek@intel.com>
Date: Tue, 11 Dec 2018 07:05:01 +0100
Subject: [PATCH 0963/1214] Revert "ASoC: Intel: board: Add support for FE
 dynamic dai for ULL"

Due to some issues with dynamic dais, we will not support them
for ULL at the momemnt, from the topology as well.

This reverts commit e7621f644c5fff60650c0bbe5741dc7e061a6a22.

Change-Id: I591f18f3a8ff808f755cc3c16ec4b8352d1d0443
Signed-off-by: Szymon Mielczarek <szymonx.mielczarek@intel.com>
Tracked-On: OAM-72117
Reviewed-by: Lewandowski, Gustaw <gustaw.lewandowski@intel.com>
Tested-by: Bozek, DominikX <dominikx.bozek@intel.com>
---
 sound/soc/intel/boards/bxt_ivi_ull.c | 54 ++++++++++++++++++++++++++++++++++++
 1 file changed, 54 insertions(+)

diff --git a/sound/soc/intel/boards/bxt_ivi_ull.c b/sound/soc/intel/boards/bxt_ivi_ull.c
index ba5293d..91baeb7 100644
--- a/sound/soc/intel/boards/bxt_ivi_ull.c
+++ b/sound/soc/intel/boards/bxt_ivi_ull.c
@@ -95,6 +95,60 @@ static const struct snd_soc_dapm_route bxtp_ull_map[] = {
 
 /* broxton digital audio interface glue - connects codec <--> CPU */
 static struct snd_soc_dai_link bxtp_ull_dais[] = {
+	{
+		.name = "Bxt Audio Port 3",
+		.stream_name = "Stereo-16K SSP4",
+		.cpu_dai_name = "System Pin 3",
+		.platform_name = "0000:00:0e.0",
+		.nonatomic = 1,
+		.dynamic = 1,
+		.codec_name = "snd-soc-dummy",
+		.codec_dai_name = "snd-soc-dummy-dai",
+		.trigger = {SND_SOC_DPCM_TRIGGER_POST,
+			SND_SOC_DPCM_TRIGGER_POST},
+		.dpcm_playback = 1,
+		.dpcm_capture = 1,
+	},
+	{
+		.name = "Bxt Audio Port 4",
+		.stream_name = "5-ch SSP1",
+		.cpu_dai_name = "System Pin 4",
+		.platform_name = "0000:00:0e.0",
+		.nonatomic = 1,
+		.dynamic = 1,
+		.codec_name = "snd-soc-dummy",
+		.codec_dai_name = "snd-soc-dummy-dai",
+		.trigger = {SND_SOC_DPCM_TRIGGER_POST,
+			SND_SOC_DPCM_TRIGGER_POST},
+		.dpcm_playback = 1,
+	},
+	{
+		.name = "Bxt Audio Port 5",
+		.stream_name = "SSP2 Stream",
+		.cpu_dai_name = "System Pin 5",
+		.platform_name = "0000:00:0e.0",
+		.nonatomic = 1,
+		.dynamic = 1,
+		.codec_name = "snd-soc-dummy",
+		.codec_dai_name = "snd-soc-dummy-dai",
+		.trigger = {SND_SOC_DPCM_TRIGGER_POST,
+			SND_SOC_DPCM_TRIGGER_POST},
+		.dpcm_playback = 1,
+		.dpcm_capture = 1,
+	},
+	{
+		.name = "Bxt Audio Port 6",
+		.stream_name = "8-Ch SSP0",
+		.cpu_dai_name = "System Pin 6",
+		.platform_name = "0000:00:0e.0",
+		.nonatomic = 1,
+		.dynamic = 1,
+		.codec_name = "snd-soc-dummy",
+		.codec_dai_name = "snd-soc-dummy-dai",
+		.trigger = {SND_SOC_DPCM_TRIGGER_POST,
+			SND_SOC_DPCM_TRIGGER_POST},
+		.dpcm_capture = 1,
+	},
 	/* Probe DAI Links */
 	{
 		.name = "Bxt Compress Probe playback",
-- 
2.7.4

